pom.xml

```diff
@@ -570,22 +523,66 @@
         </executions>
       </plugin>
 
+
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>${surefire.plugin.version}</version>
         <configuration>
           <!--
             This line enables code-coverage. However, it also blows the build's permgen.
             Fix me!
           -->
-          <argLine>-XX:-UseSplitVerifier ${jacoco.agent.unit.arg}</argLine>
+          <argLine>-Xms512m -Xmx1024m -XX:MaxPermSize=512m -XX:-UseSplitVerifier ${jacoco.agent.unit.arg}</argLine>
```

Comment:
`MaxPermSize` has been deprecated with Java8. You can drop it now or later.
http://docs.oracle.com/javase/8/docs/technotes/tools/unix/java.html